diff --git a/riplpox/riplpox.py b/riplpox/riplpox.py
index 2fef84e..70daa6f 100644
--- a/riplpox/riplpox.py
+++ b/riplpox/riplpox.py
@@ -68,7 +68,7 @@ class Switch (EventMixin):
     msg.buffer_id = buffer_id
     self.connection.send(msg)
 
-  def install(self, port, match, buf = -1, idle_timeout = 0, hard_timeout = 0):
+  def install(self, port, match, buf = None, idle_timeout = 0, hard_timeout = 0):
     msg = of.ofp_flow_mod()
     msg.match = match
     msg.idle_timeout = idle_timeout
